examples/ReduxExample/src/reducers/index.js

```diff
@@ -4,27 +4,31 @@ import { NavigationActions } from 'react-navigation';
 import { AppNavigator } from '../navigators/AppNavigator';
 
 // Start with two routes: The Main screen, with the Login screen on top.
-const initialNavState = {
-  index: 1,
-  routes: [
-    { key: 'InitA', routeName: 'Main' },
-    { key: 'InitB', routeName: 'Login' },
-  ],
-};
-
-const initialAuthState = { isLoggedIn: false };
+const firstAction = AppNavigator.router.getActionForPathAndParams('Main');
+const tempNavState = AppNavigator.router.getStateForAction(firstAction);
+const secondAction = AppNavigator.router.getActionForPathAndParams('Login');
+const initialNavState = AppNavigator.router.getStateForAction(secondAction, tempNavState);
```

Comment:
I could simply leave the initial state there as comments, to clarify what this logic all turns into. Or I could add a helper method that generates a state based on an array of actions --
 e.g. `AppNavigator.router.getInitialStateForPath('Main')` which could also take an array `AppNavigator.router.getInitialStateForPath(['Main', 'Login'])`. Seems like a very common pattern that could warrant a helper method like this.